packages/mjml-core/src/types/unit.js

```diff
@@ -21,7 +21,7 @@ export default params => {
 
       this.matchers = [
         new RegExp(
-          `^(((\\d|,|\\.){1,}(${filteredUnits.map(escapeRegExp).join('|')})|0${allowAuto})( )?){${args.join(
+          `^(((-|\\d|,|\\.){1,}(${filteredUnits.map(escapeRegExp).join('|')})|0${allowAuto})( )?){${args.join(
```

Comment:
Add an option for negative unit support ?

Reply:
an option at which level ? You would like us to only allow negative values for specific attributes ?

Reply:
At the type level, it should allow only positive or positive + negative value.

mj-column width="-600px" is valid with this change, it shouldn't